### PR TITLE
#158423104 Add home-page with login form and links for password recovery and sign-up form

### PR DESCRIPTION
**Pull request by meshack-mbuvi**
#### What does this PR do?
This pull request creates home-page with login-form and links for password recovery and sign-up forms.

#### Description of Task to be completed?
At minimum, the homepage should have the following:
- login form with input fields for user credentials
- Links for password recovery and sign-up forms.

#### How should this be manually tested?
- Clone this repository
- In your terminal, `cd ride-my-way`
- `git checkout ft-sign-in-form-158423104
- Open `index.html` using your favorite browser.

#### What are the relevant pivotal tracker stories?
[#158423104] https://www.pivotaltracker.com/story/show/158423104

#### Screenshots (if appropriate)
sign-in

### CODE REVIEW

**Comment by TheSteelGuy** [comment]
Nice UI and the pr is also following the protocol. kudos

File: ui/index.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+	                </div>
+	                <div class="col-sm-12">
+						<label for="pwd"><i class="fa fa-lock fa-fw"></i>Enter password :</label>
+					    <input type="password" name="password" placeholder="Enter password" required>
+					</div>
+						<p><a href="./recover-password.html">Forgot your password?</a></p>
+					<div class="col-sm-12">
+						<button type="submit" name="submit" id="submit" class="btn-primary"><i class="fa fa-send-o fa-fw"></i>Login</button>
+					</div>	
+					<div class="col-sm-12 col-md-6" style="width:100%;clear: both;">
```
Comment (artorious) [inline ui/index.html]: Inline CSS in HTML

File: ui/index.html [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+				
+			</div>
+		</div>
+
+		<!--Login form here-->
+		<div class="row ">
+			<div class="col-sm-12 col-md-4">
+				
+			</div>
+			<div class="col-sm-12 col-md-4 text-center div-center" style="border: solid 1px;border-radius: 5px;">
```
Comment (artorious) [inline ui/index.html]: Inline CSS in HTML

**Comment by artorious** [comment]
Great responsive design and descriptive PR.
See Self/Peer assessment Guidelines section on Inline CSS in HTML.
Otherwise, Looks awesome.

File: ui/index.html (hunk 1 above)
Comment (meshack-mbuvi) [reply ui/index.html]: Thanks so much. Your recommendations have been taken care of

**Pull request merged**